libs/use-viewer/src/utils.ts

```diff
@@ -67,17 +72,70 @@ export const buildQualityOptions = (layers: MediaLayer[]) => {
     },
     streamQuality: qualities[idx],
   }));
-  return qualityOptions;
+
+  return [{ streamQuality: 'Auto' } as SimulcastQuality, ...qualityOptions];
+};
+
+export const reprojectToMapping = async (
+  viewer: View,
+  source: RemoteTrackSource,
+  audioMapping?: ViewProjectSourceMapping,
+  videoMapping?: ViewProjectSourceMapping
+) => {
+  const { audioMediaId, sourceId, videoMediaId } = source;
+
+  const mapping: ViewProjectSourceMapping[] = [];
+
+  if (audioMediaId && audioMapping) {
+    mapping.push(audioMapping);
+  }
+
+  if (videoMediaId && videoMapping) {
+    mapping.push(videoMapping);
+  }
+
+  // await unprojectRemoteTrackSource(viewer, source);
+
+  try {
+    console.log('Reproject remote track to main stream', source);
+    await viewer.project(sourceId, mapping);
+  } catch (error: unknown) {
+    console.error(`Failed to reproject remote track to main stream mapping: ${error}`);
+  }
 };
 
-export const unprojectAndRemoveRemoteTrack = async (source: RemoteTrackSource, viewer: View) => {
-  const mids = [];
-  if (source.videoMediaId) mids.push(source.videoMediaId);
-  if (source.audioMediaId) mids.push(source.audioMediaId);
-  if (mids.length === 0) return;
+export const reprojectToOriginalMapping = async (viewer: View, source: RemoteTrackSource) => {
```

Comment:
We don't need this function, after we projected the source to its own MediaStream by `addRemoteTrackAndProject` in `active` event, it's all done.